src/routes/transactionWrapper.js

```diff
@@ -1,9 +1,22 @@
 import { sequelize } from '../models';
+import handleErrors from '../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:WRAPPER';
+
+const logContext = {
+  namespace,
+};
 
 export default function transactionWrapper(originalFunction) {
   return async function wrapper(req, res, next) {
-    return sequelize.transaction(async () => {
-      const result = await originalFunction(req, res, next);
+    return sequelize.transaction(async (t) => {
+      let result;
+      try {
+        result = await originalFunction(req, res, next);
+      } catch (err) {
+        await t.rollback();
```

Comment:
This can be deleted since we are using managed transactions. At this point, the transaction is already rolled back.